test/mailers/previews/application_mailer_preview.rb

```diff
@@ -1,6 +1,7 @@
 require "factory_bot"
 
-FactoryBot.find_definitions
```

Comment:
What are the use cases where this will be called outside of dev/test? In other words, would it possible to just remove the line entirely?

Reply:
`if Rails.env.staging?`might be more explicit then?

Reply:
I went for test and dev because it depends on the factory_bot_rails gem being loaded. And the gem is in the test and dev group: https://github.com/otwcode/otwarchive/blob/4c2621eedc1ddaf07d11bf3b5f9519e1575fb917/Gemfile#L143-L148

`if Rails.env.staging?` would indeed have the same effect. I have no strong opinions towards changing it.